heron/common/src/python/utils/misc/serializer.py

```diff
@@ -19,7 +19,7 @@
 except:
   import pickle
 
-class HeronSerializer(object):
+class IHeronSerializer(object):
```

Comment:
also wondering... we don't have to even follow the naming convention from Factory method of Java